app/src/androidTest/java/com/sdp/movemeet/Chat/MessageViewHolderTest.java

```diff
@@ -0,0 +1,7 @@
+package com.sdp.movemeet.Chat;
+
+public class MessageViewHolderTest {
```

Comment:
another empty test class ???

Reply:
This was in case we wanted to implement an additional test to cover our MessageViewHolder class. Actually, we spent a lot of time building ChatActivityTest and it paid off, because this test also allows us to indirectly test our MessageViewHolder class.